pkg/daemon/update.go

```diff
@@ -594,13 +594,9 @@ func (dn *Daemon) updateSSHKeys(newUsers []ignv2_2types.PasswdUser) error {
 
 	// Keys should only be written to "/home/core/.ssh"
 	// Once Users are supported fully this should be writing to PasswdUser.HomeDir
-	if newUsers[0].Name != "core" {
-		// Double checking that we are only writing SSH Keys for user "core"
-		return fmt.Errorf("Expecting user core. Got %s instead", newUsers[0].Name)
-	}
-	sshDirPath := filepath.Join("/home", newUsers[0].Name, ".ssh")
-	// we are only dealing with the "core" User at this time, so only dealing with the first entry in Users[]
+	sshDirPath := filepath.Join("/home", "core", ".ssh")
```

Comment:
it would be super cool if we move the string `core` to a const and use it consistently across the code (less error prone as well)

Reply:
also, if this path is gonna be just this, we might just do:
```
const (
    coreUser                = "core"
    coreUserSSHPath = "/home/core/.ssh"
)
```

Reply:
that makes sense to me!

Reply:
Done.